packages/react-core/src/components/DatePicker/examples/DatePicker.md

```diff
@@ -128,3 +128,27 @@ ControlledDate = () => {
   );
 }
 ```
+
+### Controlling the DatePicker calendar state
```

Comment:
nit.  Can you please change to sentence case.
`Controlling the date picker calendar state`

Reply:
done :)